src/main/java/tic/tac/toe/Board.java

```diff
@@ -4,26 +4,40 @@
 public class Board {
   private char[] board;
   private char currentPlayerSymbol = 'X';
+
   public Board() {
-    this.board = new char[9];
+    board = new char[9];
     Arrays.fill(board, ' ');
   }
 
   public int getBoardSize() {
     return board.length;
   }
   
+  public char getCurrentPlayerSymbol() {
+    return currentPlayerSymbol;
+  }
+
   public boolean isCellEmpty(int cell) {
     return (board[cell-1] == ' ');
   }
   
+  public boolean isGameOver() {
```

Comment:
I would rename this method; what is this method telling us about `Board`?

Reply:
Renamed to isBoardFull